targets/targets.json

```diff
@@ -256,7 +256,7 @@
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-M3",
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM", "GCC_CR", "IAR"],
-        "extra_labels": ["NXP", "LPC176X"],
+        "extra_labels": ["NXP", "LPC176X", "UBLOX_MODEM"],
```

Comment:
Please check the comment below. We need this extra label. We are not voilating any rule here, its exctly like LWIP have its own set of lwip-eth->arch->target fashion.